tests/fv3_conf/fv3_slurm.IN

```diff
@@ -3,40 +3,36 @@
 #SBATCH -o out
 #SBATCH --account=@[ACCNR]
 #SBATCH --qos=@[QUEUE]
-#SBATCH --partition=@[PARTITION]
```

Comment:
This is a temporary file created from fv3_slurm.IN_MACHINE, maybe we can delete it in next commit.

Reply:
removed the temporary fv3_slurm.IN